docs/embedding/sdk/questions.md

```diff
@@ -62,20 +62,20 @@ export default function App() {
+| `isSaveEnabled`         | boolean                                                              | (optional, default: `true`) Whether users can save the question.                                                                                                                                                                                                                                                                                                                                                                                   |
+| `onBeforeSave`          | `() => void`                                                         | (optional) A callback function that triggers before saving. Only relevant when `isSaveEnabled = true`.                                                                                                                                                                                                                                                                                                                                             |
+| `onSave`                | `() => void`                                                         | (optional) A callback function that triggers when a user saves the question. Only relevant when `isSaveEnabled = true`.                                                                                                                                                                                                                                                                                                                            |
+| `plugins`               | `{ mapQuestionClickActions: Function }` or null                      | Additional mapper function to override or add drill-down menu.                                                                                                                                                                                                                                                                                                                                                                                     |
+| `questionId`            | number or string                                                     | (required) The ID of the question. This is either:<br>- The numerical ID when accessing a question link, e.g., `http://localhost:3000/question/1-my-question` where the ID is `1`.<br>- The `entity_id` key of the question object. You can find a question's Entity ID in the info panel when viewing a question.<br>- `new` to show the notebook editor for creating new questions. `isSaveEnabled` must be `true` to allow saving the question. |
```

Comment:
Updates the `questionId` to required and replaces the line on `undefined` to `new`